doc/tutorial/basic/index.md

```diff
@@ -9,7 +9,9 @@ This is a step-by-step tutorial where we start with the setup of a Scala.js sbt
 
 To go through this tutorial, you will need to [download & install sbt](https://www.scala-sbt.org/1.x/docs/Setup.html). Note that no prior sbt knowledge (only a working installation) is required to follow the tutorial.
 
-You will also need to [download & install Node.js](https://nodejs.org/en/download/).
+You will also need to [download & install Node.js](https://nodejs.org/en/download/) and install ```jsdom``` with:
+
+    npm install jsdom
 
 To run the complete tutorial application, you will also need to install jsdom as explained [below](#supporting-the-dom).
```

Comment:
Isn't the new addition a duplicate of this sentence?

Reply:
Yes, you are right. But maybe have it here explicitly makes sense and drop the later one. At least I was stumbling over it missing it and costed me quite some time to find it.

Reply:
This may be the case, but then we should better elaborate. We need to make it clear that `npm install jsdom` must be executed in the directory of the project (which, at this point, has not been created).

Reply:
Mmh, Is this true? I don’t remember that I did this. I am not so used to Node.JS and thought jsdom is a packet installed globally for Node.js.